docs/api/instruments/hp/index.rst

```diff
@@ -11,3 +11,5 @@ This section contains specific documentation on the Hewlett Packard instruments
 
    hp33120A
    hp34401A
+   hp3478A
+
```

Comment:
This file is not part of your this pull request and it is actually part of the master branch.
Again, something wrong with rebase ?